### PR TITLE
Add Slovenia

### PR DESCRIPTION
**Pull request by stefanb**

### CODE REVIEW

File: config/regions.yaml [hunk 1]
```diff
@@ -219,6 +219,10 @@ regions:
       name: Scotland
       flag: 🏴󠁧󠁢󠁳󠁣󠁴󠁿
       osmcha.id: 2c2d94e5-29bc-4eaa-a5e2-119df5d6b649
+    slovenia:
+      name: Slovenia
+      flag: 🇸🇮
+      # osmcha.id: TODO
```
Comment (stefanb) [inline config/regions.yaml]: Please add `osmcha.id` or tell me where i can find it for Slovenia.

**Comment by jbelien** [comment]
Thanks a lot @stefanb ! I'll take care of the last steps in the next few days 👍

**Comment by jbelien** [comment]
@stefanb Everything is ready but I need the permission to write in your repository to be able to finish the last steps.

**Comment by stefanb** [comment]
@jbelien, if you have the write access to this repository you should also have the write permission on this PR's branch in my fork. Have you tried it?

**Comment by jbelien** [comment]
> @jbelien, if you have the write access to this repository you should also have the write permission on this PR's branch in my fork. Have you tried it?

No, sorry that's incorrect.
This PR is based on https://github.com/openstreetmap-si/osm-welcome-tool/tree/add-slovenia branch ; I need write permission on that repository to push my changes.

**Comment by stefanb** [comment]
Ah, yes, I forgot the detail that it wasn't my personal fork. I have sent you the collaborator invite in that fork.

**Comment by jbelien** [comment]
🚀 Deployed: https://welcome.osm.be/europe/slovenia/list

List will be automatically updated tomorrow!

**Comment by stefanb** [comment]
Thanks, @jbelien !

**Pull request merged**